compile issue

A fresh clone from github and then executed ./build.sh --jar via bash.
```
Tim@Tim-PC MINGW64 /f/Dokumente/Minecraft/Development/Compile/Paper Developemnt/Paper (master)
$ ./build.sh --jar
Downloading unmapped vanilla jar...
Applying class mappings...
Applying member mappings...
Creating remapped jar...
Installing remapped jar...
Extracting NMS classes...
Decompiling classes...
Applying CraftBukkit patches to NMS...
Patching Block.java < nms-patches/Block.patch
Patching BlockButtonAbstract.java < nms-patches/BlockButtonAbstract.patch
Patching BlockCactus.java < nms-patches/BlockCactus.patch
Patching BlockCake.java < nms-patches/BlockCake.patch
Patching BlockCauldron.java < nms-patches/BlockCauldron.patch
Patching BlockChorusFlower.java < nms-patches/BlockChorusFlower.patch
Patching BlockCocoa.java < nms-patches/BlockCocoa.patch
Patching BlockCommand.java < nms-patches/BlockCommand.patch
Patching BlockCrops.java < nms-patches/BlockCrops.patch
Patching BlockDaylightDetector.java < nms-patches/BlockDaylightDetector.patch
Patching BlockDiodeAbstract.java < nms-patches/BlockDiodeAbstract.patch
Patching BlockDispenser.java < nms-patches/BlockDispenser.patch
Patching BlockDoor.java < nms-patches/BlockDoor.patch
Patching BlockDragonEgg.java < nms-patches/BlockDragonEgg.patch
Patching BlockDropper.java < nms-patches/BlockDropper.patch
Patching BlockEnderPortal.java < nms-patches/BlockEnderPortal.patch
Patching BlockFire.java < nms-patches/BlockFire.patch
Patching BlockFlowing.java < nms-patches/BlockFlowing.patch
Patching BlockGrass.java < nms-patches/BlockGrass.patch
Patching BlockIce.java < nms-patches/BlockIce.patch
Patching BlockJukeBox.java < nms-patches/BlockJukeBox.patch
Patching BlockLeaves.java < nms-patches/BlockLeaves.patch
Patching BlockLever.java < nms-patches/BlockLever.patch
Patching BlockMinecartDetector.java < nms-patches/BlockMinecartDetector.patch
Patching BlockMinecartTrackAbstract.java < nms-patches/BlockMinecartTrackAbstract.patch
Patching BlockMobSpawner.java < nms-patches/BlockMobSpawner.patch
Patching BlockMonsterEggs.java < nms-patches/BlockMonsterEggs.patch
Patching BlockMushroom.java < nms-patches/BlockMushroom.patch
Patching BlockMycel.java < nms-patches/BlockMycel.patch
Patching BlockNetherWart.java < nms-patches/BlockNetherWart.patch
Patching BlockOre.java < nms-patches/BlockOre.patch
Patching BlockPiston.java < nms-patches/BlockPiston.patch
Patching BlockPlant.java < nms-patches/BlockPlant.patch
Patching BlockPortal.java < nms-patches/BlockPortal.patch
Patching BlockPoweredRail.java < nms-patches/BlockPoweredRail.patch
Patching BlockPressurePlateAbstract.java < nms-patches/BlockPressurePlateAbstract.patch
Patching BlockPressurePlateBinary.java < nms-patches/BlockPressurePlateBinary.patch
Patching BlockPressurePlateWeighted.java < nms-patches/BlockPressurePlateWeighted.patch
Patching BlockPumpkin.java < nms-patches/BlockPumpkin.patch
Patching BlockRedstoneLamp.java < nms-patches/BlockRedstoneLamp.patch
Patching BlockRedstoneOre.java < nms-patches/BlockRedstoneOre.patch
Patching BlockRedstoneTorch.java < nms-patches/BlockRedstoneTorch.patch
Patching BlockRedstoneWire.java < nms-patches/BlockRedstoneWire.patch
Patching BlockReed.java < nms-patches/BlockReed.patch
Patching BlockSapling.java < nms-patches/BlockSapling.patch
Patching BlockSkull.java < nms-patches/BlockSkull.patch
Patching BlockSnow.java < nms-patches/BlockSnow.patch
Patching BlockSoil.java < nms-patches/BlockSoil.patch
Patching BlockStationary.java < nms-patches/BlockStationary.patch
Patching BlockStem.java < nms-patches/BlockStem.patch
Patching BlockTrapdoor.java < nms-patches/BlockTrapdoor.patch
Patching BlockTripwire.java < nms-patches/BlockTripwire.patch
Patching BlockTripwireHook.java < nms-patches/BlockTripwireHook.patch
Patching BlockVine.java < nms-patches/BlockVine.patch
Patching BlockWaterLily.java < nms-patches/BlockWaterLily.patch
Patching BossBattleServer.java < nms-patches/BossBattleServer.patch
Patching ChatBaseComponent.java < nms-patches/ChatBaseComponent.patch
Patching ChatModifier.java < nms-patches/ChatModifier.patch
Patching Chunk.java < nms-patches/Chunk.patch
Patching ChunkProviderServer.java < nms-patches/ChunkProviderServer.patch
Patching ChunkRegionLoader.java < nms-patches/ChunkRegionLoader.patch
Patching ChunkSection.java < nms-patches/ChunkSection.patch
Patching CommandBlockListenerAbstract.java < nms-patches/CommandBlockListenerAbstract.patch
Patching CommandExecute.java < nms-patches/CommandExecute.patch
Patching CommandGamemode.java < nms-patches/CommandGamemode.patch
Patching CommandGamerule.java < nms-patches/CommandGamerule.patch
Patching CommandSpreadPlayers.java < nms-patches/CommandSpreadPlayers.patch
Patching CommandTp.java < nms-patches/CommandTp.patch
Patching Container.java < nms-patches/Container.patch
Patching ContainerAnvil.java < nms-patches/ContainerAnvil.patch
Patching ContainerBeacon.java < nms-patches/ContainerBeacon.patch
Patching ContainerBrewingStand.java < nms-patches/ContainerBrewingStand.patch
Patching ContainerChest.java < nms-patches/ContainerChest.patch
Patching ContainerDispenser.java < nms-patches/ContainerDispenser.patch
Patching ContainerEnchantTable.java < nms-patches/ContainerEnchantTable.patch
Patching ContainerFurnace.java < nms-patches/ContainerFurnace.patch
Patching ContainerHopper.java < nms-patches/ContainerHopper.patch
Patching ContainerHorse.java < nms-patches/ContainerHorse.patch
Patching ContainerMerchant.java < nms-patches/ContainerMerchant.patch
Patching ContainerPlayer.java < nms-patches/ContainerPlayer.patch
Patching ContainerWorkbench.java < nms-patches/ContainerWorkbench.patch
Patching CraftingManager.java < nms-patches/CraftingManager.patch
Patching CrashReport.java < nms-patches/CrashReport.patch
Patching DataPaletteBlock.java < nms-patches/DataPaletteBlock.patch
Patching DataWatcher.java < nms-patches/DataWatcher.patch
Patching DedicatedServer.java < nms-patches/DedicatedServer.patch
Patching DispenseBehaviorItem.java < nms-patches/DispenseBehaviorItem.patch
Patching DispenseBehaviorProjectile.java < nms-patches/DispenseBehaviorProjectile.patch
Patching DispenserRegistry.java < nms-patches/DispenserRegistry.patch
Patching DragonControllerManager.java < nms-patches/DragonControllerManager.patch
Patching Enchantment.java < nms-patches/Enchantment.patch
Patching EnchantmentFrostWalker.java < nms-patches/EnchantmentFrostWalker.patch
Patching EnchantmentThorns.java < nms-patches/EnchantmentThorns.patch
Patching EnderDragonBattle.java < nms-patches/EnderDragonBattle.patch
Patching Entity.java < nms-patches/Entity.patch
Patching EntityAgeable.java < nms-patches/EntityAgeable.patch
Patching EntityAnimal.java < nms-patches/EntityAnimal.patch
Patching EntityAreaEffectCloud.java < nms-patches/EntityAreaEffectCloud.patch
Patching EntityArmorStand.java < nms-patches/EntityArmorStand.patch
Patching EntityArrow.java < nms-patches/EntityArrow.patch
Patching EntityBoat.java < nms-patches/EntityBoat.patch
Patching EntityChicken.java < nms-patches/EntityChicken.patch
Patching EntityCow.java < nms-patches/EntityCow.patch
Patching EntityCreature.java < nms-patches/EntityCreature.patch
Patching EntityCreeper.java < nms-patches/EntityCreeper.patch
Patching EntityDamageSourceIndirect.java < nms-patches/EntityDamageSourceIndirect.patch
Patching EntityEgg.java < nms-patches/EntityEgg.patch
Patching EntityEnderCrystal.java < nms-patches/EntityEnderCrystal.patch
Patching EntityEnderDragon.java < nms-patches/EntityEnderDragon.patch
Patching EntityEnderman.java < nms-patches/EntityEnderman.patch
Patching EntityEnderPearl.java < nms-patches/EntityEnderPearl.patch
Patching EntityExperienceOrb.java < nms-patches/EntityExperienceOrb.patch
Patching EntityFallingBlock.java < nms-patches/EntityFallingBlock.patch
Patching EntityFireball.java < nms-patches/EntityFireball.patch
Patching EntityFireworks.java < nms-patches/EntityFireworks.patch
Patching EntityFishingHook.java < nms-patches/EntityFishingHook.patch
Patching EntityGhast.java < nms-patches/EntityGhast.patch
Patching EntityHanging.java < nms-patches/EntityHanging.patch
Patching EntityHorse.java < nms-patches/EntityHorse.patch
Patching EntityHuman.java < nms-patches/EntityHuman.patch
Patching EntityInsentient.java < nms-patches/EntityInsentient.patch
Patching EntityIronGolem.java < nms-patches/EntityIronGolem.patch
Patching EntityItem.java < nms-patches/EntityItem.patch
Patching EntityItemFrame.java < nms-patches/EntityItemFrame.patch
Patching EntityLargeFireball.java < nms-patches/EntityLargeFireball.patch
Patching EntityLeash.java < nms-patches/EntityLeash.patch
Patching EntityLightning.java < nms-patches/EntityLightning.patch
Patching EntityLiving.java < nms-patches/EntityLiving.patch
Patching EntityMinecartAbstract.java < nms-patches/EntityMinecartAbstract.patch
Patching EntityMinecartCommandBlock.java < nms-patches/EntityMinecartCommandBlock.patch
Patching EntityMinecartContainer.java < nms-patches/EntityMinecartContainer.patch
Patching EntityMonster.java < nms-patches/EntityMonster.patch
Patching EntityMushroomCow.java < nms-patches/EntityMushroomCow.patch
Patching EntityOcelot.java < nms-patches/EntityOcelot.patch
Patching EntityPainting.java < nms-patches/EntityPainting.patch
Patching EntityPig.java < nms-patches/EntityPig.patch
Patching EntityPlayer.java < nms-patches/EntityPlayer.patch
Patching EntityPotion.java < nms-patches/EntityPotion.patch
Patching EntityProjectile.java < nms-patches/EntityProjectile.patch
Patching EntityRabbit.java < nms-patches/EntityRabbit.patch
Patching EntitySheep.java < nms-patches/EntitySheep.patch
Patching EntityShulkerBullet.java < nms-patches/EntityShulkerBullet.patch
Patching EntitySilverfish.java < nms-patches/EntitySilverfish.patch
Patching EntitySkeleton.java < nms-patches/EntitySkeleton.patch
Patching EntitySlice.java < nms-patches/EntitySlice.patch
Patching EntitySlime.java < nms-patches/EntitySlime.patch
Patching EntitySmallFireball.java < nms-patches/EntitySmallFireball.patch
Patching EntitySnowman.java < nms-patches/EntitySnowman.patch
Patching EntitySpectralArrow.java < nms-patches/EntitySpectralArrow.patch
Patching EntitySpider.java < nms-patches/EntitySpider.patch
Patching EntityThrownExpBottle.java < nms-patches/EntityThrownExpBottle.patch
Patching EntityTippedArrow.java < nms-patches/EntityTippedArrow.patch
Patching EntityTNTPrimed.java < nms-patches/EntityTNTPrimed.patch
Patching EntityTracker.java < nms-patches/EntityTracker.patch
Patching EntityTrackerEntry.java < nms-patches/EntityTrackerEntry.patch
Patching EntityVillager.java < nms-patches/EntityVillager.patch
Patching EntityWither.java < nms-patches/EntityWither.patch
Patching EntityWitherSkull.java < nms-patches/EntityWitherSkull.patch
Patching EntityWolf.java < nms-patches/EntityWolf.patch
Patching EntityZombie.java < nms-patches/EntityZombie.patch
Patching ExpirableListEntry.java < nms-patches/ExpirableListEntry.patch
Patching Explosion.java < nms-patches/Explosion.patch
Patching FoodMetaData.java < nms-patches/FoodMetaData.patch
Patching HandshakeListener.java < nms-patches/HandshakeListener.patch
Patching IChunkProvider.java < nms-patches/IChunkProvider.patch
Patching IDataManager.java < nms-patches/IDataManager.patch
Patching IInventory.java < nms-patches/IInventory.patch
Patching InventoryCrafting.java < nms-patches/InventoryCrafting.patch
Patching InventoryCraftResult.java < nms-patches/InventoryCraftResult.patch
Patching InventoryEnderChest.java < nms-patches/InventoryEnderChest.patch
Patching InventoryHorseChest.java < nms-patches/InventoryHorseChest.patch
Patching InventoryLargeChest.java < nms-patches/InventoryLargeChest.patch
Patching InventoryMerchant.java < nms-patches/InventoryMerchant.patch
Patching InventorySubcontainer.java < nms-patches/InventorySubcontainer.patch
Patching IRecipe.java < nms-patches/IRecipe.patch
Patching ItemArmor.java < nms-patches/ItemArmor.patch
Patching ItemBoat.java < nms-patches/ItemBoat.patch
Patching ItemBow.java < nms-patches/ItemBow.patch
Patching ItemBucket.java < nms-patches/ItemBucket.patch
Patching ItemChorusFruit.java < nms-patches/ItemChorusFruit.patch
Patching ItemDye.java < nms-patches/ItemDye.patch
Patching ItemFireball.java < nms-patches/ItemFireball.patch
Patching ItemFishingRod.java < nms-patches/ItemFishingRod.patch
Patching ItemFlintAndSteel.java < nms-patches/ItemFlintAndSteel.patch
Patching ItemHanging.java < nms-patches/ItemHanging.patch
Patching ItemLeash.java < nms-patches/ItemLeash.patch
Patching ItemMapEmpty.java < nms-patches/ItemMapEmpty.patch
Patching ItemMinecart.java < nms-patches/ItemMinecart.patch
Patching ItemMonsterEgg.java < nms-patches/ItemMonsterEgg.patch
Patching ItemRecord.java < nms-patches/ItemRecord.patch
Patching ItemSnowball.java < nms-patches/ItemSnowball.patch
Patching ItemStack.java < nms-patches/ItemStack.patch
Patching ItemWaterLily.java < nms-patches/ItemWaterLily.patch
Patching ItemWorldMap.java < nms-patches/ItemWorldMap.patch
Patching JsonList.java < nms-patches/JsonList.patch
Patching LoginListener.java < nms-patches/LoginListener.patch
Patching MerchantRecipe.java < nms-patches/MerchantRecipe.patch
Patching MethodProfiler.java < nms-patches/MethodProfiler.patch
Patching MinecraftServer.java < nms-patches/MinecraftServer.patch
Patching MobEffectList.java < nms-patches/MobEffectList.patch
Patching MobSpawnerAbstract.java < nms-patches/MobSpawnerAbstract.patch
Patching NameReferencingFileConverter.java < nms-patches/NameReferencingFileConverter.patch
Patching NetworkManager.java < nms-patches/NetworkManager.patch
Patching PacketDataSerializer.java < nms-patches/PacketDataSerializer.patch
Patching PacketPlayInBlockPlace.java < nms-patches/PacketPlayInBlockPlace.patch
Patching PacketPlayInCloseWindow.java < nms-patches/PacketPlayInCloseWindow.patch
Patching PacketStatusListener.java < nms-patches/PacketStatusListener.patch
Patching PathfinderGoalBreakDoor.java < nms-patches/PathfinderGoalBreakDoor.patch
Patching PathfinderGoalBreed.java < nms-patches/PathfinderGoalBreed.patch
Patching PathfinderGoalDefendVillage.java < nms-patches/PathfinderGoalDefendVillage.patch
Patching PathfinderGoalEatTile.java < nms-patches/PathfinderGoalEatTile.patch
Patching PathfinderGoalHorseTrap.java < nms-patches/PathfinderGoalHorseTrap.patch
Patching PathfinderGoalHurtByTarget.java < nms-patches/PathfinderGoalHurtByTarget.patch
Patching PathfinderGoalMakeLove.java < nms-patches/PathfinderGoalMakeLove.patch
Patching PathfinderGoalNearestAttackableTarget.java < nms-patches/PathfinderGoalNearestAttackableTarget.patch
Patching PathfinderGoalNearestAttackableTargetInsentient.java < nms-patches/PathfinderGoalNearestAttackableTargetInsentient.patch
Patching PathfinderGoalOwnerHurtByTarget.java < nms-patches/PathfinderGoalOwnerHurtByTarget.patch
Patching PathfinderGoalOwnerHurtTarget.java < nms-patches/PathfinderGoalOwnerHurtTarget.patch
Patching PathfinderGoalPanic.java < nms-patches/PathfinderGoalPanic.patch
Patching PathfinderGoalSit.java < nms-patches/PathfinderGoalSit.patch
Patching PathfinderGoalTame.java < nms-patches/PathfinderGoalTame.patch
Patching PathfinderGoalTarget.java < nms-patches/PathfinderGoalTarget.patch
Patching PathfinderGoalTargetNearestPlayer.java < nms-patches/PathfinderGoalTargetNearestPlayer.patch
Patching PlayerChunk.java < nms-patches/PlayerChunk.patch
Patching PlayerChunkMap.java < nms-patches/PlayerChunkMap.patch
Patching PlayerConnection.java < nms-patches/PlayerConnection.patch
Patching PlayerInteractManager.java < nms-patches/PlayerInteractManager.patch
Patching PlayerInventory.java < nms-patches/PlayerInventory.patch
Patching PlayerList.java < nms-patches/PlayerList.patch
Patching PortalTravelAgent.java < nms-patches/PortalTravelAgent.patch
Patching PropertyManager.java < nms-patches/PropertyManager.patch
Patching RecipeArmorDye.java < nms-patches/RecipeArmorDye.patch
Patching RecipeBookClone.java < nms-patches/RecipeBookClone.patch
Patching RecipeFireworks.java < nms-patches/RecipeFireworks.patch
Patching RecipeMapClone.java < nms-patches/RecipeMapClone.patch
Patching RecipeRepair.java < nms-patches/RecipeRepair.patch
Patching RecipesBanner.java < nms-patches/RecipesBanner.patch
Patching RecipesFurnace.java < nms-patches/RecipesFurnace.patch
Patching RecipeTippedArrow.java < nms-patches/RecipeTippedArrow.patch
Patching RecipiesShield.java < nms-patches/RecipiesShield.patch
Patching RegionFile.java < nms-patches/RegionFile.patch
Patching RemoteControlCommandListener.java < nms-patches/RemoteControlCommandListener.patch
Patching ScoreboardServer.java < nms-patches/ScoreboardServer.patch
Patching SecondaryWorldServer.java < nms-patches/SecondaryWorldServer.patch
Patching ShapedRecipes.java < nms-patches/ShapedRecipes.patch
Patching ShapelessRecipes.java < nms-patches/ShapelessRecipes.patch
Patching Slot.java < nms-patches/Slot.patch
Patching SlotFurnaceResult.java < nms-patches/SlotFurnaceResult.patch
Patching SpawnerCreature.java < nms-patches/SpawnerCreature.patch
Patching StatisticManager.java < nms-patches/StatisticManager.patch
Patching TileEntity.java < nms-patches/TileEntity.patch
Patching TileEntityBanner.java < nms-patches/TileEntityBanner.patch
Patching TileEntityBeacon.java < nms-patches/TileEntityBeacon.patch
Patching TileEntityBrewingStand.java < nms-patches/TileEntityBrewingStand.patch
Patching TileEntityChest.java < nms-patches/TileEntityChest.patch
Patching TileEntityCommand.java < nms-patches/TileEntityCommand.patch
Patching TileEntityContainer.java < nms-patches/TileEntityContainer.patch
Patching TileEntityDispenser.java < nms-patches/TileEntityDispenser.patch
Patching TileEntityEndGateway.java < nms-patches/TileEntityEndGateway.patch
Patching TileEntityFurnace.java < nms-patches/TileEntityFurnace.patch
Patching TileEntityHopper.java < nms-patches/TileEntityHopper.patch
Patching TileEntityNote.java < nms-patches/TileEntityNote.patch
Patching TileEntityPiston.java < nms-patches/TileEntityPiston.patch
Patching TileEntitySign.java < nms-patches/TileEntitySign.patch
Patching TileEntitySkull.java < nms-patches/TileEntitySkull.patch
Patching UserCache.java < nms-patches/UserCache.patch
Patching Village.java < nms-patches/Village.patch
Patching VillageSiege.java < nms-patches/VillageSiege.patch
Patching World.java < nms-patches/World.patch
Patching WorldBorder.java < nms-patches/WorldBorder.patch
Patching WorldData.java < nms-patches/WorldData.patch
Patching WorldGenGroundBush.java < nms-patches/WorldGenGroundBush.patch
Patching WorldGenMegaTreeAbstract.java < nms-patches/WorldGenMegaTreeAbstract.patch
Patching WorldGenRegistration.java < nms-patches/WorldGenRegistration.patch
Patching WorldGenVillagePieces.java < nms-patches/WorldGenVillagePieces.patch
Patching WorldManager.java < nms-patches/WorldManager.patch
Patching WorldMap.java < nms-patches/WorldMap.patch
Patching WorldNBTStorage.java < nms-patches/WorldNBTStorage.patch
Patching WorldServer.java < nms-patches/WorldServer.patch
Rebuilding Forked projects....
/f/Dokumente/Minecraft/Development/Compile/Paper Developemnt/Paper/Spigot /f/Dokumente/Minecraft/Development/Compile/Paper Developemnt/Paper
Resetting Spigot-API to Bukkit...
HEAD is now at cd53bd9 SPIGOT-1934: Expand EnderDragon API - adds dragon phases
  Applying patches to Spigot-API...
Applying: POM Changes
Applying: Spigot Timings
Applying: Add PlayerItemDamageEvent
Applying: BungeeCord Support
Applying: Add Arrow API
Applying: Add Particle API
Applying: Define EntitySpawnEvent and SpawnerSpawnEvent
Applying: Entity Mount and Dismount Events
Applying: InventoryClickEvent getClickedInventory
Applying: Added getAllSessionData() to the Conversation API.
Applying: Catch Conversation API Errors
Applying: Player Collision API
Applying: Expand Boolean Prompt Values
Applying: Add Getter for Entity Invulnerability
Applying: Add respawn API.
Applying: Fix Plugin Message API Disconnects
Applying: Fix Tab Completion for Some Commands
Applying: Implement Locale Getter for Players
Applying: Add support for fetching hidden players
Applying: Silenceable Lightning API
Applying: Remove deprecation on some player lookup methods
Applying: Add Score.isScoreSet()Z API.
Applying: Add PlayerSpawnLocationEvent.
Applying: Ease ClassLoader Deadlocks Where Possible
Applying: Fix slow tab complete for some commands.
Applying: Undeprecate Player#updateInventory()V
Applying: Added isUnbreakable and setUnbreakable to ItemMeta
Applying: BungeeCord Chat API
Applying: Add restart API.
  Patches applied cleanly to Spigot-API
Resetting Spigot-Server to CraftBukkit...
HEAD is now at 3ad80d9 CraftBukkit $ Mi, 23. Mrz 2016 19:45:11
  Applying patches to Spigot-Server...
Applying: POM Changes
Applying: Skeleton API Implementations
Applying: mc-dev imports
Applying: Obfuscation Helpers
Applying: Spigot Configuration
Applying: Crop Growth Rates
Applying: Merge tweaks and configuration
Applying: Async Operation Catching
Applying: View Distance
Applying: Spigot Timings
Applying: Fix Mob Spawning Relative to View Distance
Applying: Handle Null Tile Entities
Applying: Item Despawn Rate
Applying: Entity Activation Range
Applying: Metrics
Applying: PlayerItemDamageEvent
Applying: Prevent NPE in CraftSign
Applying: Entity Tracking Ranges
Applying: Thread Naming and Tweaks
Applying: Close Unloaded Save Files
Applying: Remove -o Option
Applying: Recipe Deconstruction
Applying: Implement Arrow API
Applying: Hopper Customisations
Applying: Firework Meta Crash Fix
Applying: Allow Disabling of Command Logging
Applying: Allow Disabling of Command TabComplete
Applying: Configurable Messages
Applying: Allow Disabling of Random Lighting Updates
Applying: Properly Close Inventories
Applying: Disallow Interaction With Self
Applying: Entity Mount and Dismount Events
Applying: Prevent Ghost Players Caused by Plugins
Applying: Plug World Unload Memory Leak
Applying: Player Collision API
Applying: Fully Disable Snooper When Not Required
Applying: Add Getter for Entity Invulnerability
Applying: Cap Minimum Player Speed
Applying: Call EntityChangeBlockEvent for Fire Arrows hitting TNT
Applying: Allow Disabling of 1.6.3 Structure Saving
Applying: Don't Special Case X Move Value
Applying: Implement respawn API.
Applying: Arrow Despawn Rate
Applying: Watchdog Thread.
Applying: Fix some chunks not being sent to the client
Applying: Fix Broken Async Chat
Applying: Fire PreLogin Events in Offline Mode
Applying: BungeeCord Support
Applying: Allow Disabling Zombie Villager Aggression
Applying: Configurable Amount of Netty Threads
Applying: Prevent Mineshaft Saving
Applying: Log Cause of Unexpected Exceptions
Applying: Particle API
Applying: Save ticks lived to nbttag
Applying: Add Option to Nerf Mobs from Spawner's
Applying: Warn if PermGen may be insufficient
Applying: Disable Connected Check on setScoreboard
Applying: Add Late Bind Option
Applying: Allow statistics to be disabled/forced
Applying: Fix ItemStack Unbreakable Code
Applying: Try and Debug Crash Reports Crashing
Applying: Improve AutoSave Mechanism
Applying: Catch stalling on corrupted map data / NBT arrays.
Applying: Allow toggling of ZombiePigmen spawning in portal blocks
Applying: Highly Optimized Tick Loop
Applying: Configurable Ping Sample Size
Applying: Add Optional Tick Shuffling
Applying: Implement Locale Getter for Players
Applying: Cap Entity Collisions
Applying: Fix dispensing bone meal not having the correct data value
Applying: Spam Filter Exclusions
Applying: Add Option to Silence CommandBlock Console
Applying: Add support for fetching hidden players
Applying: Allow Disabling Creative Item Filter
Applying: Cap Channel Registrations
Applying: Allow vanilla commands to be the main version of a command
Applying: Implement Silenceable Lightning API
Applying: Use one PermissibleBase for all Command Blocks
Applying: Prevent hoppers from loading chunks
Applying: Guard Entity List
Applying: Fix ConcurrentModificationException while being idle kicked in a vehicle
Applying: Cancellable WitherSkull potion effect
Applying: Descriptive kick reasons instead of Nope!
Applying: Configurable dragon death and wither spawn sounds
Applying: Display 'Spigot' in client crashes, server lists and Mojang stats
Applying: Treat Bungee as Online Mode
Applying: Add Conversion Message
Applying: Properly cancel fishing event. Fixes BUKKIT-5396
Applying: Print Stack on InternalException
Applying: Use Offline Player Data Once if Required.
Applying: Use Provided Case for Non Existent Offline Players
Applying: Check for blank OfflinePlayer Names
Applying: Fix Player Banning
Applying: Fix ban expire dates.
Applying: Correct Ban Expiration
Applying: Convert Horses owner to UUID
Applying: Add Score.isScoreSet()Z API.
Applying: Log null TileEntity Owner
Applying: Don't special case 'invalid' usernames for UUIDs.
Applying: Convert player skulls async
Applying: Prevent NoClassDefError crash and notify on crash
Applying: Check Skull canPlace
Applying: Fix race condition that could kill connections before they were initiated
Applying: Configurable UserCache cap
Applying: Implement PlayerSpawnLocationEvent.
Applying: Cap Objective Score Length
Applying: Process conversation input on the main thread. Fixes BUKKIT-5611
Applying: Configurable save-on-stop-only for UserCache
Applying: Prevent Unbounded IntCache Growth
Applying: Add damager to the unhandled error
Applying: Cross World Entity Teleportation
Applying: Limit block placement/interaction packets
Applying: Better item validation
Applying: Further Seed Customisation
Applying: Disable ResourceLeakDetector
Applying: Add More Information to session.lock Errors
Applying: Safer JSON Loading
Applying: Add CommandLine EULA Flag
Applying: Fix for enchanting table wiping meta when placing stacked items
Applying: Don't spawn bonus ocelots when plugins spawn ocelots
Applying: Prevent a crash involving attributes
Applying: Make "moved wrongly" limit configurable
Applying: Fix Null Tile Entity Worlds
Applying: Make "moved too quickly" limit configurable
Applying: Apply NBTReadLimiter to more things.
Applying: Allow Attribute Capping.
Applying: Only fetch an online UUID in online mode
Applying: Filter attribute modifiers which cause the attribute to go out of its range
Applying: Plug WorldMap Memory Leak
Applying: Fix Corrupted Trapped Chest
Applying: Remove uneeded validation
Applying: Add Hunger Config Values
Applying: Make debug logging togglable.
Applying: Skip invalid enchants in CraftMetaItem
Applying: Limit TNT Detonations per tick
Applying: Added isUnbreakable and setUnbreakable to ItemMeta
Applying: Configurable Hanging Tick
Applying: Refactor ItemDoor Place
Applying: BungeeCord Chat API
Applying: Use Standard List for EntitySlices.
Applying: Allow Capping (Tile)Entity Tick Time.
Applying: Use FastMatches for ItemStack Dirty Check
Applying: Use CLQ for main thread tasks.
Applying: map decoration fix
Applying: Use Map for getPlayer(String) lookup.
Applying: Clear Packet Queue on Disconnect
Applying: More effecient RegionFile zero'ing
Applying: Implement SpawnerSpawnEvent.
Applying: Optimize Navigation Listener
  Patches applied cleanly to Spigot-Server
/f/Dokumente/Minecraft/Development/Compile/Paper Developemnt/Paper
./applyPatches.sh: line 12: cd: /f/Dokumente/Minecraft/Development/Compile
Developemnt/Paper/Spigot/Spigot-API: No such file or directory
./applyPatches.sh: line 16: cd: /f/Dokumente/Minecraft/Development/Compile
Developemnt/Paper: No such file or directory
fatal: destination path 'Paper-API' already exists and is not an empty directory.
./applyPatches.sh: line 20: cd: /f/Dokumente/Minecraft/Development/Compile
Developemnt/Paper/Paper-API: No such file or directory
Resetting Paper-API to Spigot-API...
fatal: ambiguous argument 'upstream/upstream': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
  Applying patches to Paper-API...
fatal: could not open '/f/Dokumente/Minecraft/Development/Compile
Developemnt/Paper/Spigot-API-Patches/*.patch' for reading: No such file or directory
  Something did not apply cleanly to Paper-API.
  Please review above details and finish the apply then
  save the changes with rebuildPatches.sh
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Child module F:\Dokumente\Minecraft\Development\Compile\Paper Developemnt\Paper\Paper-Server of F:\Dokumente\Minecraft\Development\Compile\Paper Developemnt\Paper\pom.xml does not exist @
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project com.destroystokyo.paper:paper-parent:dev-SNAPSHOT (F:\Dokumente\Minecraft\Development\Compile\Paper Developemnt\Paper\pom.xml) has 1 error
[ERROR]     Child module F:\Dokumente\Minecraft\Development\Compile\Paper Developemnt\Paper\Paper-Server of F:\Dokumente\Minecraft\Development\Compile\Paper Developemnt\Paper\pom.xml does not exist
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

--- a/applyPatches.sh
+++ b/applyPatches.sh
@@ -43,7 +43,7 @@ basedir=$basedir/Spigot
 applyPatch ../Bukkit Spigot-API HEAD && applyPatch ../CraftBukkit Spigot-Server patched
 # Move out of Spigot
 popd
-basedir=$(dirname $basedir)
+basedir=$(dirname "$basedir")
 
 # Apply paper
 applyPatch Spigot/Spigot-API Paper-API HEAD && applyPatch Spigot/Spigot-Server Paper-Server HEAD

